ruka/metallb/mlb-l2.yaml

```diff
@@ -10,4 +10,4 @@ data:
     - name: default
       protocol: layer2
       addresses:
-      - 139.229.135.150-139.229.135.249
+      - 139.229.135.150-139.229.135.199
```

Comment:
It looks like the original IP range is still in IPAM (the DHCP pool seems to be missing). The revised range also appears to overlap with the DHCP pool: https://github.com/lsst-it/lsst-itconf/blob/master/hieradata/site/ls/role/foreman.yaml#L29 and one of them needs to be adjusted.  The DHCP pool is probably much larger than is necessary so it may make more sense to shrink it.

Reply:
This one will need more digging. I'll create another branch for both repos once I figure out a simple solution